qa/tasks/cephfs/test_admin.py

```diff
@@ -2210,6 +2210,7 @@ def test_with_health_warn_oversize_cache(self):
         self.negtest_ceph_cmd(args=f'fs fail {self.fs.name}',
                               retval=1, errmsgs=errmsg)
         self.run_ceph_cmd(f'fs fail {self.fs.name} --yes-i-really-mean-it')
+        self.mount_a.umount_wait(force=True, lazy=True)
```

Comment:
it shouldn't be necessary to force or use lazy. The MDS will come back. You need to:

```suggestion
        self.fs.wait_for_daemons()
```

So the MDS has a chance to become active. Then unmount.

Reply:
> The MDS will come back.

It won't without us allowing it to. So `self.fs.wait_for_daemons()` should be preceded by `self.fs.set_joinable()`